Integrations/integration-AWS-IAM.yml

```diff
@@ -1024,6 +1024,71 @@ script:
         except Exception as e:
             return raise_error(e)
 
+
+    def get_account_password_policy(args):
+        try:
+            client = aws_session(
+                    roleArn = args.get('roleArn'),
+                    roleSessionName = args.get('roleSessionName'),
+                    roleSessionDuration = args.get('roleSessionDuration'),
+                )
+
+            response = client.get_account_password_policy()
+            data = (response['PasswordPolicy'])
+
+            raw = json.loads(json.dumps(response['PasswordPolicy'], cls=DatetimeEncoder))
+            ec = {'AWS.IAM.PasswordPolicy': raw}
+            return create_entry('AWS IAM Account Password Policy', data, ec)
+
+        except Exception as e:
+            return raise_error(e)
+
+
+    def update_account_password_policy(args):
+        try:
+            client = aws_session(
+                    roleArn=args.get('roleArn'),
+                    roleSessionName=args.get('roleSessionName'),
+                    roleSessionDuration=args.get('roleSessionDuration'),
+                )
+            try:
+                response = client.get_account_password_policy()
+                kwargs = response['PasswordPolicy']
+            except client.exceptions.NoSuchEntityException as e:
+                kwargs = {}
+
+            # ExpirePasswords is part of the response but cannot be included
+            # in the request
+            if 'ExpirePasswords' in kwargs:
+                kwargs.pop('ExpirePasswords')
+
+            if args.get('minimumPasswordLength'):
+                kwargs.update({'MinimumPasswordLength': int(args.get('minimumPasswordLength'))})
+            if args.get('requireSymbols'):
+                kwargs.update({'RequireSymbols': bool(args.get('requireSymbols'))})
```

Comment:
```suggestion
                kwargs.update({'RequireSymbols': True if args.get('requireSymbols') == 'True' else False})
```

Reply:
Generally, I use this method as it allows for a default value in the event one is not passed to it. It's not necessary to do it this way however.